### PR TITLE
[FIX] l10n_pe: Add domain 01 for export refunds

### PR DESCRIPTION
**Pull request by luistorresm**
In commit 355250d, the domain was adjusted to automatically select the invoice document type for customers with a RUC and limit others to the "boleta" document type. This generally holds true, except for export invoices requiring the 01 document type. This commit addresses the current restriction, allowing the selection of document type 01 other than "credit note boleta."

[Enterprise PR related](https://github.com/odoo/enterprise/pull/56465)

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by fvz-odoo** [comment]
Hello @luistorresm 

Thanks for the PR. This is now working correctly as well as the linked PR https://github.com/odoo/enterprise/pull/56465. You can go ahead and check @RicGR98, thanks!

Cheers!

Fernando

**Review by RicGR98 — COMMENTED** [review]
Thanks for the PR ! :)
Could you please link the entreprise PR in the PR description ? :)

**Comment by luistorresm** [comment]
> Thanks for the PR ! :) Could you please link the entreprise PR in the PR description ? :)

Done

**Comment by RicGR98** [comment]
Runbot is still red

**Comment by fvz-odoo** [comment]
Hello @RicGR98 

Ready now. 

Cheers!

Fernando

Thanks @luistorresm

**Comment by RicGR98** [comment]
Thanks for the changes :)

Is this not linked to any task ?

**Comment by fvz-odoo** [comment]
> Thanks for the changes :)
> 
> Is this not linked to any task ?

Hello @RicGR98 

No, we don't... do you want me to create it?

Cheers!

Fernando

**Comment by RicGR98** [comment]
@fvz-odoo no need, thanks :)

**Comment by RicGR98** [comment]
@fvz-odoo 

Just have to wait for the final review by @oco-odoo but perhaps @smetl or @jco-odoo can check it too :)

**Review by smetl — APPROVED** [review]
@robodoo r+

**Comment by smetl** [comment]
@robodoo r+

**Pull request closed without merging**
